packages/environment-check/src/archive/index.ts

```diff
@@ -35,14 +44,14 @@ export async function archiveProject(
                     .replace(':', '')
                     .substring(0, 14)}.zip`;
             }
-            const targetPath = join(dirname(projectRoot), targetName);
-            const writeStream = createWriteStream(targetPath);
+            const targetFolder = targetPath ? join(targetPath, targetName) : join(dirname(projectRoot), targetName);
```

Comment:
@zdravko-georgiev, the content of the changes looks good, but I'm a bit confused about the naming of variables:
- `targetPath` sounds like the complete path with filename
- `targetFolder` sounds like the folder the archive should be stored in

as far as I can see, the names are used the opposite here

Reply:
Thanks Klaus, done with https://github.com/SAP/open-ux-tools/pull/894/commits/b94dddd6cba9aafa1b686c0a6539604a0b609ba0